src/Controller/Admin/Base/AbstractImportBaseController.php

```diff
@@ -89,39 +94,54 @@ protected function createJsonResponse(bool $success, string $message, int $statu
      */
     protected function replaceIfExists(AbstractElement $element): int
     {
-        $oldElement = $this->repository->getByPath('/' . $element->getFullPath());
+        $oldElement = $this->repository->getByPath($element->getFullPath());
         if (null !== $oldElement) {
             if ($this->overwrite) {
-                $oldElement->delete();
-                $element->save(['versionNote' => 'overwritten by pimcore-import-export-bundle']);
+                if (0 === $element->getId() || null === $element->getId() || $oldElement->getId() === $element->getId()) {
+                    $oldElement->delete();
```

Comment:
Wenn das alte Element gelöscht wird, werden aber immer noch auch alle Kinder gelöscht. Ich hab's ausprobiert und es funktioniert nicht.

Reply:
Solved, by refactoring the code